apps/server/src/modules/files-storage/interface/interfaces.ts

```diff
@@ -0,0 +1,21 @@
+import { Readable } from 'stream';
+import type { DownloadFileParams, PreviewParams } from '../controller/dto';
+import { FileRecord } from '../entity';
+
+export interface GetFileResponse {
+	data: Readable;
+	etag: string | undefined;
```

Comment:
```suggestion
	etag?: string;
```
I prefer this way of marking it optional, but I don't know if we have a decision on that...